cluster-autoscaler/simulator/clustersnapshot/store/delta.go

```diff
@@ -437,6 +439,16 @@ func (snapshot *DeltaSnapshotStore) DraSnapshot() *drasnapshot.Snapshot {
 	return snapshot.draSnapshot
 }
 
+// CsiSnapshot returns the CSI snapshot.
+func (snapshot *DeltaSnapshotStore) CsiSnapshot() *csisnapshot.Snapshot {
+	return snapshot.csiSnapshot
+}
+
+// CSINodes returns the CSI node lister for this snapshot.
+func (snapshot *DeltaSnapshotStore) CSINodes() fwk.CSINodeLister {
```

Comment:
Could you move this method to where the other scheduler-framework-specific methods are (after `DeviceClassResolver()`)?